Dockerfile

```diff
@@ -33,6 +33,13 @@ RUN apt-get update && apt-get -y dist-upgrade && \
     && mkdir -p /var/run/dbus \
     && mkdir -p /etc/chromium/policies/managed /etc/chromium/policies/recommended \
     && mkdir /tmp/.X11-unix && chmod 1777 /tmp/.X11-unix && chown root /tmp/.X11-unix
+    
+ #Install Widevine Component for Netflix
```

Comment:
Very, *very* specific gripe here: the tabbing is kind of off for this, plus I believe it could be worded a bit better.

I believe it should be something like this:
```
# Install Widevine component for Chromium
```

Since Widevine isn't only used by Netflix.

Reply:
Done! I've only tested it on Netflix so I wasn't too sure about the other websites.